docs/us.md

```diff
@@ -0,0 +1,225 @@
+# Update system consensus rules
+
+## Prerequisites
+
+* [Update Mechanism](https://cardanodocs.com/cardano/update-mechanism/).
+
+## Overview
+Update system gives ability to developers to propose updates of software applications and
+ability to users to vote for updates to decide which one will be accepted.
+
+This document describes a global part of update system consensus rules, as well difference between a local part.
+Global consensus rules is a part of block processing, which checks validity of update payload of blocks and update 
+corresponding state when blocks are applied/rolled back. 
+We first describe global part and then describe how local part differs.
+
+## Problem definition
+TBA
+
+## Update system model
+
+### Poll and decision agreement rules
+
+Each stakeholder can send update proposal in the blockchain,
```

Comment:
I'm not sure if it's written below, but I'd already say that update proposal can appear in blockchain only if it has some votes for it from stakeholders (there is a threshold, you know).

Reply:
Frankly, I am not sure it should be mentioned here, this rule is kind of just in case, to avoid an influence of bad men. I wouldn't mention it here...